projects/forge-angular/src/lib/list/list-value-accessor.directive.ts

```diff
@@ -0,0 +1,98 @@
+import { Directive, ElementRef, forwardRef, HostListener, Renderer2, StaticProvider } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { IListComponent, IListItemComponent, IListItemSelectEventData, LIST_ITEM_CONSTANTS } from '@tylertech/forge';
+
+export const LIST_VALUE_ACCESSOR: StaticProvider = {
+  provide: NG_VALUE_ACCESSOR,
+  useExisting: forwardRef(() => ListValueAccessor),
+  multi: true
+};
+
+@Directive({
+  selector: `forge-list[formControlName],forge-list[formControl],forge-list[ngModel]`,
+  providers: [LIST_VALUE_ACCESSOR]
+})
+export class ListValueAccessor implements ControlValueAccessor {
+  @HostListener('forge-list-item-select', ['$event'])
+  public listItemSelect(event: CustomEvent<IListItemSelectEventData>): void {
+    const { detail: { value } } =  event;
+    let listValue: any[] = this._elementRef.nativeElement.selectedValue;
```

Comment:
We should try to avoid using `any` as much as possible — try setting this to `unknown` if the value type doesn't matter here